plugins/jobtype/src/azkaban/jobtype/SparkJobArg.java

```diff
@@ -0,0 +1,36 @@
+package azkaban.jobtype;
+
+public enum SparkJobArg {
+
+  MASTER("master", "--master", "yarn-cluster"), // just to trick the eclipse formatter
+  SPARK_JARS("jars", "--jars", "./lib/*"), //
+  EXECUTION_JAR("execution.jar", null, null), //
+  EXECUTION_CLASS("execution.class", "--class", null), //
+  NUM_EXECUTORS("num.executors", "--num-executors", "2"), //
+  EXECUTOR_CORES("executor.cores", "--executor-cores", "1"), //
+  QUEUE("queue", "--queue", "marathon"), //
+  DRIVER_MEMORY("driver.memory", "--driver-memory", "2g"), //
+  EXECUTOR_MEMORY("executor.memory", "--executor-memory", "1g"), //
+  PARAMS("params", "null", ""), //
+  SPARK_CONF_PREFIX("spark.conf.", "--conf", ""), //
+  SPARK_DRIVER_PREFIX("spark.driver.", "--", ""), //
+  DRIVER_JAVA_OPTIONS("spark.driver.driver-java-options", "--driver-java-options", ""),
+  SPARK_FLAG_PREFIX("spark.flag.", "--", ""); // --help, --verbose, --supervise, --version
```

Comment:
Why are we not providing a full coverage of all Spark-submit command line options?